doc/schema/schema.md

```diff
@@ -70,7 +81,7 @@ Information and diagnostics for each `posterior` sample, provided by the inferen
 * `max_energy_error`
 
 ### `posterior_predictive`
-Posterior predictive samples p(y|y) corresponding to the posterior predictive distribution evaluated at the `observed_data`. Samples should match with `posterior` ones and each variable should have a counterpart in `observed_data`. The `observed_data` counterpart variable may have a different name.
+Posterior predictive samples p(y|y) corresponding to the posterior predictive distribution evaluated at the `observed_data`. Samples should match with `posterior` ones and its variables should match `observed_data` variables. The `observed_data` counterpart variable may have a different name.
```

Comment:
I followed this until the last sentence.  How would the observed data counterpart variable have a different name?  And if it *did* have a different name, in what way would it be a counterpart?

An example might clarify this, but IMO we should not rely on a notion of "counterpart" if we don't have a definition, in code, of how one could test two variables to see if they are counterparts.

Reply:
I think a code returning `True` if two variables can be counterparts is possible, however, it would never be able to reject variables having the same shape by chance. 

The "same variable/different name" already works in ArviZ without problem (see `data_pairs` argument in `plot_ppc`), I think only objects coming from Stan world need it.